### ISSUE TITLE
Not working with offline-content

### ISSUE THREAD

**Issue by robwent** [mozilla/wp-offline-shell#104]
When the offline-content plugin is disabled, offline-shell seems to work as expected, but I understood from the plugin page that they should be able to work together.

If both plugins are enabled then only offline-content items are cached by the browser, nothing happens with shell files.

If I then disable offline-content the shell files are cached as expected.

If I then try and turn offline-content back on, I get a fatal error:

    Fatal error: Uncaught Error: Class 'Mozilla\WP_SW_Manager' not found in xxxx/public/wp-content/plugins/offline-content/class-wp-offline-content-plugin.php

If I then deactivate offline-shell, offline-content enables without any issues and I can re-enable offline-shell

I've tried toggling the different options in both plugins but can't seem to get them working well together.

Am I missing something obvious?

**Comment by mykmelez** [comment]
@delapuente Are the wp-offline-shell and wp-offline-content plugins still maintained? If not, perhaps we should mark them as discontinued here in GitHub and remove them from the WordPress Plugin Directory so folks don't continue to rely on them.

**Comment by delapuente** [comment]
You're right. They are not actively maintained and we can point it there but they should continue working for the WP versions they already did. I'll take the proper actions to inform about it. Thank you.
